lib/models/User.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:anxiety_go/models/difficulty.dart';
+
+class User {
+  String firstName;
+  String lastName;
+  String email;
+  String password;
+  Difficulty difficultyLevel;
+
+  User({
+    required this.firstName,
+    required this.lastName,
+    required this.email,
+    required this.password,
+    required this.difficultyLevel,
+  });
+
+  Map<String, Object> toJSON() {
```

Comment:
Make this method static

Reply:
Heya @markhallak you can't make that method static as it requires an instance of the object to exist to be able to build the map with the data from that instance.

Error message from Dart is
```
Instance members can't be accessed from a static method.
Try removing the reference to the instance member, or removing the keyword 'static' from the method.
```

Did you mean something else?